File: aula_11-04/2_calculadora_imc.py
# O código abaixo é uma tentativa de criar uma calculadora de Índice de Massa Corporal (IMC).
# No entanto, ele está incompleto e contém erros.
# Modifique o código para que ele solicite ao usuário seu peso em quilogramas e altura em metros.
# Seu programa deve calcular e imprimir o IMC do usuário seguindo a fórmula IMC = peso / (altura ** 2).
# Além disso, deve classificar o resultado em: Baixo peso, Peso normal, Sobrepeso ou Obesidade, 
# baseando-se nos valores de IMC.

peso = float(input("Digite seu peso em kg: "))
altura = float(input("Digite sua altura em metros: "))

imc = 0  # Cálculo do IMC precisa ser corrigido

# Classificação do IMC (faltam informações)
if imc < 18.5:
    classificacao = "Baixo peso"
elif imc < 30:
    classificacao = "Sobrepeso"
else:
    classificacao = "Obesidade"

print(f"Seu IMC é {imc} e sua classificação é: {classificacao}")

File: aula_12-04/2_calculadora-imc.cpp
#include <iostream>
using namespace std;

int main() {
    double peso, altura, imc;
    string classificacao;

    cout << "Digite seu peso em kg: ";
    cin >> peso;
    cout << "Digite sua altura em metros: ";
    cin >> altura;

    imc = 0 // Cálculo do IMC precisa ser corrigido

    if (imc < 18.5) {
        classificacao = "Baixo peso";
    } else if (imc < 30) {
        classificacao = "Sobrepeso";
    } else {
        classificacao = "Obesidade";
    }

    cout << "Seu IMC é " << imc << " e sua classificação é: " << classificacao << endl;

    return 0;
}
